Index: scikit_build_core-0.11.3/pyproject.toml
===================================================================
--- scikit_build_core-0.11.3.orig/pyproject.toml
+++ scikit_build_core-0.11.3/pyproject.toml
@@ -26,7 +26,6 @@ classifiers = [
     "Programming Language :: Python :: 3.11",
     "Programming Language :: Python :: 3.12",
     "Programming Language :: Python :: 3.13",
-    "Programming Language :: Python :: 3.14",
     "Development Status :: 4 - Beta",
     "Typing :: Typed",
 ]
