Index: ledfx-2.0.108/PKG-INFO
===================================================================
--- ledfx-2.0.108.orig/PKG-INFO
+++ ledfx-2.0.108/PKG-INFO
@@ -18,7 +18,6 @@ Requires-Dist: openrgb-python>=0.2.15
 Requires-Dist: paho-mqtt>=1.6.1
 Requires-Dist: psutil>=5.9.7
 Requires-Dist: pyserial>=3.5
-Requires-Dist: pystray>=0.19.5
 Requires-Dist: python-rtmidi>=1.5.8
 Requires-Dist: requests>=2.31.0
 Requires-Dist: sacn>=1.9.0
Index: ledfx-2.0.108/ledfx/__main__.py
===================================================================
--- ledfx-2.0.108.orig/ledfx/__main__.py
+++ ledfx-2.0.108/ledfx/__main__.py
@@ -168,22 +168,6 @@ def parse_args():
         type=str,
     )
 
-    group = parser.add_mutually_exclusive_group()
-
-    group.add_argument(
-        "--tray",
-        dest="tray",
-        action="store_true",
-        help="Force LedFx system tray icon",
-    )
-
-    group.add_argument(
-        "--no-tray",
-        dest="no_tray",
-        action="store_true",
-        help="Force no LedFx system tray icon",
-    )
-
     parser.add_argument(
         "--offline",
         dest="offline_mode",
@@ -269,30 +253,7 @@ def main():
         _LOGGER.warning("Steering LedFx into a brick wall.")
         div_by_zero = 1 / 0
 
-    if (args.tray or currently_frozen()) and not args.no_tray:
-        # If pystray is imported on a device that can't display it, it explodes. Catch it
-        try:
-            import pystray
-        except Exception as Error:
-            msg = f"Unable to create tray icon. Error: {Error}. Try launching LedFx via --no-tray option."
-            _LOGGER.critical(msg)
-            # Exit with code 3 to indicate that there was an error creating the tray icon.
-            sys.exit(3)
-
-        from PIL import Image
-
-        icon_location = get_icon_path("tray.png")
-
-        icon = pystray.Icon(
-            "LedFx", icon=Image.open(icon_location), title="LedFx"
-        )
-    else:
-        icon = None
-
-    if icon:
-        icon.run(setup=entry_point)
-    else:
-        entry_point()
+    entry_point()
 
 
 def entry_point(icon=None):
Index: ledfx-2.0.108/pyproject.toml
===================================================================
--- ledfx-2.0.108.orig/pyproject.toml
+++ ledfx-2.0.108/pyproject.toml
@@ -17,7 +17,6 @@ dependencies = [
     "paho-mqtt>=1.6.1",
     "psutil>=5.9.7",
     "pyserial>=3.5",
-    "pystray>=0.19.5",
     "python-rtmidi>=1.5.8",
     "requests>=2.31.0",
     "sacn>=1.9.0",
